_maps/map_files/generic/CentCom_skyrat_z2.dmm

```diff
@@ -1509,11 +1509,11 @@
 /turf/open/floor/iron/dark,
 /area/cruiser_dock)
 "aqu" = (
-/obj/item/toy/plush/arcplush{
-	desc = "A small plushie that resembles a, quote, nerd - according to a crudely scribbled tag stapled onto the plush.";
-	name = "nerd plushie"
-	},
 /obj/structure/fluff/beach_umbrella/engine,
+/obj/item/paper{
+	info = "Unfortunately, due to unforseen circumstances, the Nerd Plushie will now have to go on extended vacation to avoid all of what is occuring here. You may be able to find her in the Hyperion system on Layenia, however. You may keep her umbrella, though, she may be back. - A.D.X.";
```

Comment:
adding a reference to another server in your removal PR is the fast-track to getting it stalemated